.github/workflows/run_detekt_and_unit_tests.yml

```diff
@@ -28,18 +28,19 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-gradle-
 
-      - name: Run Detekt on template
-        working-directory: ./template
+        # TODO: Add new step for template-compose in https://github.com/nimblehq/android-templates/issues/333
+      - name: Run Detekt on template-xml
+        working-directory: ./template-xml
         run: ./gradlew detekt
 
-      - name: Archive Detekt reports on template
+      - name: Archive Detekt reports on template-xml
         uses: actions/upload-artifact@v2
         with:
           name: DetektReportsTemplate
-          path: template/build/reports/detekt/
+          path: template-xml/build/reports/detekt/
 
-      - name: Run unit tests with Kover on template
-        working-directory: ./template
+      - name: Run unit tests with Kover on template-xml
+        working-directory: ./template-xml
         run: ./gradlew koverMergedHtmlReport
 
       - name: Archive code coverage reports on template
```

Comment:
I think we might have to update some more names as discussed 🙏🏼 
<img width="617" alt="Screenshot 2565-11-18 at 11 48 23" src="https://user-images.githubusercontent.com/13492460/202619336-2ae95443-4629-4a07-97fa-549671166cc7.png">